src/main/java/staffconnect/logic/parser/DeleteMeetingCommandParser.java

```diff
@@ -0,0 +1,48 @@
+package staffconnect.logic.parser;
+
+import static java.util.Objects.requireNonNull;
+import static staffconnect.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static staffconnect.logic.parser.CliSyntax.PREFIX_MEETING_INDEX;
+
+import staffconnect.commons.core.index.Index;
+import staffconnect.logic.commands.DeleteMeetingCommand;
+import staffconnect.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new DeleteMeetingCommand object
+ */
+public class DeleteMeetingCommandParser implements Parser<DeleteMeetingCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the DeleteMeetingCommand
+     * and returns a DeleteMeetingCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public DeleteMeetingCommand parse(String args) throws ParseException {
+        requireNonNull(args);
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args, PREFIX_MEETING_INDEX);
+
+        Index personIndex;
+
+        try {
+            personIndex = ParserUtil.parseIndex(argMultimap.getPreamble());
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    DeleteMeetingCommand.MESSAGE_USAGE), pe);
+        }
+
+        argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_MEETING_INDEX);
+        Index meetingIndex;
+        try {
+            meetingIndex = ParserUtil.parseIndex(argMultimap.getValue(PREFIX_MEETING_INDEX).get());
+        } catch (ParseException pe) {
+            throw new ParseException(String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                    DeleteMeetingCommand.MESSAGE_USAGE), pe);
+        }
+
+        return new DeleteMeetingCommand(personIndex, meetingIndex);
+    }
+
+}
```

Comment:
There is a bug when using the `meeting-delete` command when missing the meeting index arguments.

E.g:
```
meeting 1 d/Blank s/11/11/1111 11:11
meeting-delete 1
```

Can try:
`argMultimap.verifyNoDuplicatePrefixesFor(PREFIX_MEETING_INDEX);`

Reply:
The bug has been fixed, though the cause was different from what you pointed out!

Reply:
mybad i copied the wrong line! but thanks for the fix